### DISCUSSION TITLE
Slightly misleading questions in Unit 1 Quiz?

### DISCUSSION THREAD

**Discussion by richplant** [pykale/transparentML#31] [Suggestions] [answered]
Couple of questions that are a little hard to answer definitively:

* Data Science is not a field of Machine Learning - Given answer: False

Not only is this a double negative, I also disagree with the answer: data science isn't a field of machine learning, ML is a sub-field of general data science.

* A well defined loss function will always find the best model f(x) - Given answer: False

Trick question alert! It is of course not true that a given loss function **always** will determine the optimal solution, it certainly **may** do so. Nevertheless, it's hard to say that given alternate conditions any given objective won't deliver the optimum.

**Comment by Schobs** [comment, marked as answer]
1)  I agree

2) If "best model" = local minima then I guess it's True, if "best model" means global minima then it's False. I think the spirit of the question is "global minima" in respect to the user's expectations.
